config/c-code/tests.yml.j2

```diff
@@ -414,6 +414,10 @@ jobs:
           name: manylinux_${{ matrix.image }}_wheels.zip
       - name: Restore pip cache permissions
         run: sudo chown -R $(whoami) ${{ steps.pip-cache-default.outputs.dir }}
+{% if with_future_python %}
+      - name: Prevent publishing wheels for unreleased Python versions
+        run: VER=$(echo '%(future_python_version)s') | tr -d .) && rm -f wheelhouse/*-cp${VER}*.whl
```

Comment:
The `.)` bit breaks with a bash syntax error:   it needs to be quoted, e.g. `tr -d ".)"`.